batcher/aligned-sdk/src/sdk.rs

```diff
@@ -308,7 +308,7 @@ async fn _verify_proof_onchain(
 ) -> Result<bool, errors::VerificationError> {
     let contract_address = match chain {
         Chain::Devnet => "0x1613beB3B2C4f22Ee086B2b38C1476A3cE7f78E8",
-        Chain::Holesky => "0x58F280BeBE9B34c9939C3C39e0890C81f163B623",
+        Chain::Holesky => "0x0584313310bD52B77CF0b81b350Ca447B97Df5DF",
```

Comment:
So now the fork would point to our own contract by default right? We should take note about this if we redeploy the contract in the future